modules/jcommunity/install/install.php

```diff
@@ -234,6 +234,9 @@ protected function insertUsers(InstallHelpers $helpers, EntryPoint $entryPoint,
 
         $dao = jDao::get($daoSelector, $dbProfile);
         foreach($usersToInsert as $userData) {
+            if (empty($userData)) {
```

Comment:
the array should not content empty object, this is not relevant.

however, probably the test `if (!$usersToInsert) {` is wrong and should be `if (!is_array($usersToInsert)) {`